### ISSUE TITLE
Hot splash screen for the site

### ISSUE THREAD

**Issue by JedLJRobertson** [SE701ProjectGroup4/Flatmate-Management-System#58]
**User story**
As a user who is not logged into any account, when I arrive at the site, I should see a splash screen explaining the app.

There should be buttons for me to sign up and log in.

**Acceptance criteria**
- [ ] Looks good
- [ ] Succinct explanation of what the app does
- [ ] Login button leading to login page
- [ ] Registration button leading to registration page

**Additional context**
**Out of scope: Login/Registration pages** (see #23 )
This is not fully blocked by #23 , as most of the work can be done before hand.
